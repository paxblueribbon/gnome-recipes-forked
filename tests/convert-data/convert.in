1	g
1	kg
1	lb
1	oz
1	l
1	dl
1	ml
1	fl oz
1	pt
1	qt
1	gal
1	cup
1	tbsp
1	tsp
1	tsp
1	st
