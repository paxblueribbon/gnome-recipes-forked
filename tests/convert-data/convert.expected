INPUT '1	g'
DIMENSION 2
METRIC 1 g
IMPERIAL 0.035274 oz

INPUT '1	kg'
DIMENSION 2
METRIC 1000 g
IMPERIAL 35.274 oz

INPUT '1	lb'
DIMENSION 2
METRIC 453.592 g
IMPERIAL 16 oz

INPUT '1	oz'
DIMENSION 1
METRIC 1 oz
IMPERIAL 1 oz

INPUT '1	l'
DIMENSION 1
METRIC 1000 ml
IMPERIAL 202.884 tsp

INPUT '1	dl'
DIMENSION 1
METRIC 100 ml
IMPERIAL 20.2884 tsp

INPUT '1	ml'
DIMENSION 1
METRIC 1 ml
IMPERIAL 0.202884 tsp

INPUT '1	fl oz'
DIMENSION 1
METRIC 29.5735 ml
IMPERIAL 6 tsp

INPUT '1	pt'
DIMENSION 1
METRIC 473.176 ml
IMPERIAL 96 tsp

INPUT '1	qt'
DIMENSION 1
METRIC 946.353 ml
IMPERIAL 192 tsp

INPUT '1	gal'
DIMENSION 1
METRIC 3785.41 ml
IMPERIAL 768 tsp

INPUT '1	cup'
DIMENSION 1
METRIC 236.59 ml
IMPERIAL 48 tsp

INPUT '1	tbsp'
DIMENSION 1
METRIC 14.79 ml
IMPERIAL 3 tsp

INPUT '1	tsp'
DIMENSION 1
METRIC 4.92892 ml
IMPERIAL 1 tsp

INPUT '1	tsp'
DIMENSION 1
METRIC 4.92892 ml
IMPERIAL 1 tsp

INPUT '1	st'
DIMENSION 2
METRIC 6350.29 g
IMPERIAL 224 oz

